scripts/local-build.sh

```diff
@@ -2,6 +2,6 @@
 set -e
 # show the commands we execute
 set -o xtrace
-export IMAGE_NAME="kbase/relation_engine_api:0.0.5"
+export IMAGE_NAME="kbase/relation_engine_api:local_build"
```

Comment:
Here's the approach I've been using lately: https://github.com/kbase/search_api2/blob/develop/scripts/docker_deploy

I set a canonical version in one place in `/VERSION` and refer to that when building and deploying.

Reply:
I'll put in an issue to migrate to that syntax in this repo.